logical-comaprison-operators.js

```diff
@@ -6,35 +6,35 @@ TASK 1:
  // Have fun! // 😃
 ********************************************************************************/
 
-const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE --> true
 
-const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE -->
+const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE --> true
 
-const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE -->true
 
-const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE -->false
 
-const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE -->
+const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE -->true
 
-const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE -->
+const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE --> true
 
-const exp7 = "true" == true; // TODO: ADD YOUR EVALUATION HERE -->
+const exp7 = "true" == true; // TODO: ADD YOUR EVALUATION HERE -->true
 
-const exp8 = "false" == false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp8 = "false" == false; // TODO: ADD YOUR EVALUATION HERE -->true
 
-const exp9 = NaN === NaN; // TODO: ADD YOUR EVALUATION HERE -->
+const exp9 = NaN === NaN; // TODO: ADD YOUR EVALUATION HERE -->true
```

Comment:
This expression should be `false`, `NaN` means not a number, so technically not a number will always be not equal to any other number, because it's unknown by design.